snitun/multiplexer/core.py

```diff
@@ -31,7 +30,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 PEER_TCP_TIMEOUT = 90
-
+MAX_QUEUED_MESSAGES = 12000
```

Comment:
This is a bit odd since the previous queue accepted 12000 messages which means we could end up holding a lot in memory. Thats a lot of back pressure allowed.